.gitignore

```diff
@@ -39,6 +39,8 @@ _ReSharper.*
 *.AssemblyInfo.cs
 *.csprojAssemblyReference.cache
 *.AssemblyInfoInputs.cache
+sample-database-template/*.bak
+sample-database-template/*.nuspec
```

Comment:
Isn't that too aggressive since we're reusing an existing source control folder to store the bak/nuspec temporarily?  Won't that exclude any changes to the powershell down the road?

Reply:
should I remove gitignore?